.github/workflows/test.yml

```
@@ -13,14 +16,12 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ~/memgraph
-        key: cache-memgraph-v1.3.0-docker-image
+        key: cache-memgraph-v${{env.MG_VERSION}}-docker-image
```

```suggestion
        key: cache-memgraph-v${{ env.MG_VERSION }}-docker-image
```
I think this is the convention for using variables in Github Actions. Don't forget to change it everywhere else.